fastlane/Fastfile

```diff
@@ -8,6 +8,7 @@ skip_docs
 
 configured = false
 is_build_pr = false
+releases_domain = 'https://releases.mattermost.com'
```

Comment:
Why? This should be an env var or something, just to make sure ppl tha use the script do not have issues

Reply:
I see. I used the same S3 bucket name instead.